tests/countries/test_thailand.py

```diff
@@ -457,269 +386,148 @@ def test_new_years_eve(self):
             "2029-01-02",
         )
 
-    def test_makha_bucha(self):
```

Comment:
Here I meant the redundancy of not entirely Thai calendar holidays tests, but only testing the year 2158.